content/docs/reference/config/istio.networking.v1alpha3/index.html

```diff
@@ -1392,8 +1392,8 @@ <h2 id="HTTPMatchRequest">HTTPMatchRequest</h2>
 <p>HttpMatchRequest specifies a set of criterion to be met in order for the
 rule to be applied to the HTTP request. For example, the following
 restricts the rule to match only requests where the URL path
-starts with /ratings/v2/ and the request contains a <code>cookie</code> with value
```

Comment:
All changes in this file need to be undone and instead applied to the appropriate proto file in the istio/api repo. Once the proto is updated, this HTML file will automatically get regenerated.

Thanks.